traversability_estimation/launch/traversability_estimation.launch

```diff
@@ -4,7 +4,7 @@
     <rosparam command="load" file="$(find traversability_estimation)/config/robot.yaml"/>
     <rosparam command="load" file="$(find traversability_estimation)/config/robot_footprint_parameter.yaml"/>
     <rosparam command="load" file="$(find traversability_estimation)/config/robot_filter_parameter.yaml"/>
-    <param name="elevation_map/topic" value="grid_map"/>
+    <param name="elevation_map/load/topic" value="grid_map"/>
     <param name="elevation_map/load/path_to_bag" value="$(find traversability_estimation)/maps/elevation_map.bag"/>
     <param name="traversability_map/save/path_to_bag" value="$(find traversability_estimation)/maps/traversability_map.bag"/>
```

Comment:
You could add here setting the parameter for the saving topic of the traversability map bag.

Reply:
what do you mean? I think I did not understand your point.

Reply:
```suggestion
     <param name="traversability_map/save/path_to_bag" value="$(find traversability_estimation)/maps/traversability_map.bag"/>
     <param name="traversability_map/save/topic_name" value="traversability_map"/>
```

Reply:
oh yes, I forgot that param! Thanks :)